web/src/components/RasaWidget.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactElement, useEffect } from 'react'
```

Comment:
I think I'd rather name this `ChatBotWidget` as the technology behind it is an implementation detail and not really relevant/may not be known when looking for the code.